ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -802,6 +802,12 @@ tests:
       OCP_ARCH: amd64
       TEST_SUITE: upgrade-conformance
     workflow: openshift-upgrade-gcp-heterogeneous
+- as: e2e-agent-ibmz-sno
```

Comment:
Hi @tvardema ,
Our ABI jobs are pretty straight forward without much configurations in the job.
We enabled this from [4.15 jobs] (https://github.com/openshift/release/blob/080804ae08497ce10bc9311b6760751ddf38a4a2/ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml#L714-L719) itself and its running since then.
We enabled this as part of ABI GA for IBMZ during 4.15.